invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/ItemsSearch.js

```diff
@@ -100,46 +121,57 @@ export class ItemsSearch extends Component {
     return <div>{totalResults} results</div>;
   };
 
-  _renderResultsPerPage = (currentSize, values, onChange) => {
-    return (
-      <div>
-        <span className="results-per-page">Show</span>
-        <Dropdown
-          inline
-          compact
-          options={values}
-          value={currentSize}
-          onChange={(e, { value }) => onChange(value)}
+  _renderResultsSorting = () => {
+    return sortByValues.length ? (
+      <div className="n-per-page-sorting">
+        <span className="before">Show</span>
+        <ResultsPerPage
+          values={resultsPerPageValues}
```

Comment:
Can you capitalize that so we can identify that is a global variable and not passed as a prop?

Reply:
Done!